cmd/juju/model/configcommand.go

```diff
@@ -335,7 +335,17 @@ func (c *configCommand) getConfig(client configCommandAPI, ctx *cmd.Context) err
 		} else {
 			return errors.Errorf("key %q not found in %q model.", key, attrs["name"])
 		}
+	} else {
+		// In tabular format, don't print "cloudinit-userdata" it can be very long,
+		// instead give instructions on how to print specifically.
+		if value, ok := attrs["cloudinit-userdata"]; ok && c.out.Name() == "tabular" {
```

Comment:
Is there a const defined for this config key? I think so, we should use it

Reply:
done.